.github/workflows/post-commit.yml

```diff
@@ -17,414 +17,9 @@ on:
   pull_request:
     types: [opened, ready_for_review, synchronize]
   push:
-  schedule:
-    - cron: 30 0,12 * * *
-env:
-  FAIL_FAST: ${{ github.event_name == 'pull_request' }}
-  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3
 concurrency:
   group: ci-${{ github.event.pull_request.number || github.sha }}
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 jobs:
```

Comment:
Thanks @adoroszlai for simplifying the workflows and making the build-branch workflow avail to new contributors with this patch.  
With this change, we should also update the `Ozone CI with Githubs Actions md `documentation file, https://github.com/adoroszlai/hadoop-ozone/blob/09dfaffbd86a3adcd49402b61a251c98412c509e/.github/ci.md?plain=1#L1,  to note the the build-branch jobs are in the new `ci.yml` file that is triggered on pull through the `post-commit.yml.`  Perhaps also noting the scheduling can be enabled / disabled on forks to run twice a day, (00:30 and 12:30) and updating that the scheduling is done through the new scheduled `ci.yml`.

Reply:
Thanks for the reminder @neils-dev, updating the doc is a good suggestion.  I'll do this in a follow-up PR, because the separate doc change will require fewer checks, rather than repeating the full CI required for this change.